examples/2d_isometry/map.rs

```diff
@@ -0,0 +1,55 @@
+use std::collections::HashMap;
+
+use amethyst::core::{GlobalTransform, IsometricTransform, Parent, Transform};
+use amethyst::ecs::prelude::*;
+use amethyst::prelude::*;
+use amethyst::renderer::{SpriteRender, SpriteSheetHandle, Transparent};
+
+pub const UNIT_DIMENSIONS: (f32, f32) = (128.0, 64.0);
+const MAP_SIZE: usize = 16;
+
+// 067: Grass
+#[cfg_attr(rustfmt, rustfmt_skip)]
+const MAP: [[u8;MAP_SIZE];MAP_SIZE] = [
```

Comment:
I would've liked to see this use Tiled's format for loading the map, but I suppose this will do for now.

Reply:
Well I guess, but this PR is about isometry more than tiling.